docs/reference/resource.rst

```diff
@@ -15,12 +15,12 @@ Full example
 
     import colander
 
-    from cliquet import resource
+    import cliquet
     from cliquet import schema
     from cliquet import utils
 
 
-    class BookmarkSchema(resource.ResourceSchema):
+    class BookmarkSchema(cliquet.ResourceSchema):
```

Comment:
I suppose we could even rename ``cliquet.ResourceSchema`` to ``cliquet.Schema`` ?